cricPredict/Groups/serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+from .models import Group
+from User_profile.models import User
+
+
+class GroupSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Group
+        fields = ('name', 'privacy', 'users')
+
+    def create(self, validated_data):
+        user = User.objects.filter(username=self.context['request'].user)
```

Comment:
we should never access/send HTTP request to serializers, we could always send `request.data`to a serializer while creating an instance of the serializer.